interface-definitions/load-balancing-wan.xml.in

```diff
@@ -0,0 +1,395 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="load-balancing">
+    <properties>
+      <help>Configure load-balancing</help>
+    </properties>
+    <children>
+      <node name="wan" owner="${vyos_conf_scripts_dir}/load-balancing-wan.py">
+        <properties>
+          <help>Configure Wide Area Network (WAN) load-balancing</help>
+        </properties>
+        <children>
+          <leafNode name="disable-source-nat">
+            <properties>
+              <help>Disable source nat rules from being configured for wan load balancing</help>
```

Comment:
Let's capitalize abbreviations here and everywhere (i.e. s/wan/WAN/).

Reply:
Done